### PR TITLE
Use getTableSchema from TpchGen in TpchQueryBuilder

### PR DESCRIPTION
**Pull request by pramodsatya**

### CODE REVIEW

File: velox/exec/tests/utils/TpchQueryBuilder.cpp [hunk 1]
```diff
@@ -565,52 +566,15 @@ const std::unordered_map<std::string, std::vector<std::string>>
...
-                "o_orderkey",
-                "o_custkey",
-                "o_orderstatus",
-                "o_totalprice",
-                "o_orderdate",
-                "o_orderpriority",
-                "o_clerk",
-                "o_shippriority",
-                "o_comment"}),
+            (tpch::getTableSchema(tpch::Table::TBL_ORDERS)->names())),
```
Comment (pedroerp) [inline velox/exec/tests/utils/TpchQueryBuilder.cpp:572]: parenthesis here are unnecessary.

**Comment by majetideepak** [comment]
nice!

**Review by majetideepak — APPROVED** [review]

**Review by mbasmanova — APPROVED** [review]
@pramodsatya Nice refactor. Thanks.

**Review by aditi-pandit — APPROVED** [review]
Looks good !

**Pull request closed without merging**